src/components/gltf-model-plus.js

```diff
@@ -505,6 +505,21 @@ class GLTFHubsComponentsExtension {
       for (const componentName in ext) {
         const props = ext[componentName];
         for (const propName in props) {
+          // These components had a variant before mhc_link_type existed that just directly pointed at the node index, fix them
+          if (
+            ((componentName === "video-texture-target" && propName === "srcNode") ||
+              (componentName === "audio-target" && propName === "srcNode")) &&
+            typeof props[propName] === "number"
+          ) {
+            console.warn(
+              `Found an outdated ${componentName} 'srcNode' property, fixing. Make sure you are using the latest exporter.`
+            );
+            props[propName] = {
+              __mhc_link_type: "node",
+              index: props[propName]
+            };
+          }
```

Comment:
If traversing were free (which it's not), it'd be nice to run all the migrations we want to first and separately. If we have more of these over time, then at least we can consider extracting it to functions (like you did in the `uv-scroll` system).

Reply:
Yeah I like the idea of moving all these sorts of migrations "up" into GLTF loading. Basically making the GLTF file we wish we had before we load it, that way ll of this logic happens up front and doesn't leak elsewhere. Main downside I see of that is that its then just a big clump of random things, rather than being "near" related stuff. I could also imagine some of the transformations will be slightly more tricky to do up front. But this doesn't seem worth trying to do until we get rid of gltf-mdoel-plus and gltf-component-mappings.